guides/source/debugging_rails_applications.md

```diff
@@ -255,7 +255,8 @@ is your best companion.
 
 The debugger can also help you if you want to learn about the Rails source code
 but don't know where to start. Just debug any request to your application and
-use this guide to learn how to move from the code you have written into the underlying Rails code.
+use this guide to learn how to move from the code you have written into the
+underlying Rails code.
```

Comment:
The whole guide is wrapped to 80 lines, except for a couple of places I changed...